spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/GcpCloudSqlProperties.java

```diff
@@ -38,7 +38,7 @@
 
 	private String jdbcUrl;
 
-	private String jdbcDriver = "com.mysql.jdbc.Driver";
+	private String jdbcDriver;
```

Comment:
Do we need this property if we really stick only with what we have in the `DatabaseType` enum?

Reply:
Yes, because users might want a driver class that's not in any DatabaseType.